tests/cli/bootsys/test_service_activity.py

```diff
@@ -401,6 +401,7 @@ class TestBOSV2ActivityChecker(unittest.TestCase):
 
     def setUp(self):
         self.mock_bos_client = MagicMock()
+        self.mock_sat_session = patch('sat.cli.bootsys.service_activity.SATSession').start()
```

Comment:
I noticed that this `TestBOSV2ActivityChecker` class is missing a `tearDown` method that stops the patches. We should add that.